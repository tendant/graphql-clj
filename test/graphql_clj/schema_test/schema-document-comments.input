# this is a document comment for the type Mineral
type Mineral {
    # this is not a document comment, since it is separated
    # from the document comment by an empty line

    # This is the actual document comment for the mohs
    # field, it extends over multiple lines.
    mohs:int
    type:Type # type does not have a document comment
} # this is not part of the document comment, since it trails after a token
# but this is part of the document comment
# for the enum that follows
enum Type {
    # Gold, silver, ...
    NATIVE_ELEMENT,
    # Sapphire, hematite, ...
    OXIDE,
    # Quartz, feldspar, ...
    SILICATE
    # ... more
}

