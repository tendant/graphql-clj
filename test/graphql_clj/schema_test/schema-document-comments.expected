{:objects
 {:Mineral {:fields {:mohs {:type :int}, :type {:type :Type}}}},
 :enums {:Type {:values [:NATIVE_ELEMENT :OXIDE :SILICATE]}}}
